spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -10,10 +18,51 @@
     end
 
     context "when sending a welcome_thread notification" do
-      xit "generates the appropriate broadcast to be sent to a user"
-      xit "it sends a welcome notification for that broadcast"
-      xit "it does not send duplicate welcome notification for that broadcast"
-      xit "does not send a notification to a user who has commented in a welcome thread"
+      let(:welcome_thread_article) { create(:article, title: "Welcome Thread") }
+      let(:welcome_thread_comment) { create(:comment, commentable: welcome_thread_article, user: user) }
+      let(:article) { create(:article) }
+      let(:comment) { create(:comment, commentable: article, user: receiving_user) }
+
+      it "generates the correct broadcast type and sends the notification to the user", :aggregate_failures do
+        expect(receiving_user.notifications.count).to eq(0)
+
+        sidekiq_perform_enqueued_jobs do
```

Comment:
Question for reviewers: Is it necessary to preface `perform_enqueued_jobs` with `sidekiq` as I've done here? I have seen it primarily written as `perform_enqueued_jobs` in our specs but I have also seen it written in the way that I've written it as well.

Reply:
I'm curious about this too. I'd imagine since we replaced our delayed_job with sidekiq, we would only be using `sidekiq_perform_enqueued_jobs`. Is that right @mstruve ?

Reply:
We should only be using `sidekiq_perform_enqueued_jobs` but `perform_enqueued_jobs` will still work since it is an [ActiveJob Helper](https://api.rubyonrails.org/v5.2.3/classes/ActiveJob/TestHelper.html#method-i-perform_enqueued_jobs) and we still use active job for things like emails, we just do it using sidekiq now
```
config.active_job.queue_adapter = :sidekiq
```

Reply:
I'd say we should create an issue to remove `perform_enqueued_jobs` and establish a convention of only using the `sidekiq_` preprended one and alleviate this confusion moving forward. Working on that would be a great cool down period task, too, @juliannatetreault 😉

Reply:
Sounds like a plan @vaidehijoshi! 😎